src/data/speakers.yml

```diff
@@ -33,3 +33,17 @@
   github: https://github.com/jeromehardaway
   bio: " "
   company: " "
+- id: sethalexander
```

Comment:
Whoops. Funny thing is that is won't rebase even though it has the same content.